hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeManager.java

```diff
@@ -89,6 +89,8 @@ public class DatanodeManager {
 
   private volatile long heartbeatIntervalSeconds;
   private volatile int heartbeatRecheckInterval;
+  /** Used by {@link HeartbeatManager}. */
+  private volatile int heartbeatRecheckIntervalForMonitor;
```

Comment:
shouldAbortHeartbeatCheck should be long, not int.

Reply:
Fixed